### PR TITLE
UI

### PR DESCRIPTION
**Pull request by AW0000**
Added the UI components for the unique graph calculator page.

### CODE REVIEW

File: project/PolyPlotter/src/main/java/ch/tbz/menu/GraphInputMenu.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
+package ch.tbz.menu;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+/**
+ * Menu that handles the custom graph entering
```
Comment (vilichtSarina) [inline project/PolyPlotter/src/main/java/ch/tbz/menu/GraphInputMenu.java]: Please make the comment a complete sentence.

File: project/PolyPlotter/src/main/java/ch/tbz/menu/GraphInputMenu.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    public GraphInputMenu(int stdWidth, int stdHeight, MainFrame main) {
+        this.stdWidth = stdWidth;
+        this.stdHeight = stdHeight;
+        this.main = main;
+
+        this.setPreferredSize(new Dimension(stdWidth, stdHeight));
+        this.setLayout(new GridBagLayout());
+
+
+        GridBagConstraints gbc = new GridBagConstraints();
```
Comment (vilichtSarina) [inline project/PolyPlotter/src/main/java/ch/tbz/menu/GraphInputMenu.java]: Would prefer this setup outside of the contructors to not distract too much, WDYT? IMO an optimal solution would be a small private helper method towards the end.

**Review by vilichtSarina — CHANGES_REQUESTED** [review]
Hi Amanda, generally nice changes, just some small nits that would be nice.

File: project/PolyPlotter/src/main/java/ch/tbz/menu/GraphInputMenu.java (hunk 1 above)
Comment (AW0000) [reply project/PolyPlotter/src/main/java/ch/tbz/menu/GraphInputMenu.java]: Changed the comment.

File: project/PolyPlotter/src/main/java/ch/tbz/menu/GraphInputMenu.java (hunk 2 above)
Comment (AW0000) [reply project/PolyPlotter/src/main/java/ch/tbz/menu/GraphInputMenu.java]: Makes sense it is more readable that way, I added a helper method.

**Review by vilichtSarina — APPROVED** [review]
:)

**Pull request merged**